content/hi/docs/home/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+approvers:
+- mittalyashu
+title: कुबरनेट्स प्रलेखन
+noedit: true
+cid: docsHome
+layout: docsportal_home
+class: gridPage gridPageHome
+linkTitle: "Home"
+main_menu: true
+weight: 10
+hide_feedback: true
+menu:
+  main:
+    title: "प्रलेखन"
+    weight: 20
+    post: >
+      <p>कुबरनेट्स का उपयोग वैचारिक, ट्यूटोरियल और संदर्भ प्रलेखन के साथ करना सीखें। आप <a href="/editdocs/" data-auto-burger-exclude data-proofer-ignore>डॉक्स में योगदान</a> करने में सहायता भी कर सकते हैं!</p>
+description: >
+  कुबरनेट्स एक ओपन सोर्स container orchestration engine for automating deployment, scaling, and management of containerized applications. यह ओपन सोर्स प्रोजेक्ट को क्लाउड नेटिव कम्प्यूटिंग फाउंडेशन।
```

Comment:
> container orchestration engine for automating deployment, scaling, and management of containerized applications.

This looks like a _lot_ of English - is that how you intended it @mittalyashu ?

Reply:
I wanted to ask if I should localise these words too.

If I remember correctly, last time also we have left few words in english. Wanted to make sure how to proceed further?

Reply:
@kubernetes/sig-docs-hi-reviews thoughts?